charts/osm/values.yaml

```diff
@@ -267,12 +265,6 @@ OpenServiceMesh:
     # -- Enables SnapshotCache feature for Envoy xDS server.
     enableSnapshotCacheMode: false
 
-  #
-  # -- OSM multicluster feature configuration
-  multicluster:
```

Comment:
I suggest retaining this top level key so multicluster configs can be easily added to this root property.